flake.nix

```diff
@@ -4,8 +4,9 @@
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
     nil_crypto3 = {
-      url =
-        "git+https://github.com/NilFoundation/crypto3?submodules=1&rev=1cf64acec2f4022224cb19673854250e08a6d7c3";
+      url = "https://github.com/NilFoundation/crypto3";
+      type = "git";
+      submodules = true;
```

Comment:
good thing, that we don't need submodules anymore, as there are none left in crypto3

Reply:
Thanks, fixed